### DISCUSSION TITLE
back mouse button is linked to mouse-3 and not mouse-5

### DISCUSSION THREAD

**Discussion by brabalan** [d12frosted/homebrew-emacs-plus#685] [Q&A]
I have a Logitech MX-ergo, with two extra buttons (forward and back). While forward is mapped to mouse-4 in emacs, back is mapped to mouse-3 and thus acts as a right click. The same mouse works correctly with emacs-mac. Is there a way to distinguish these buttons in emacs-plus?

**Comment by d12frosted** [comment]
I don't have an answer for that. Probably emacs-mac made some fix. Emacs+ is just GNU Emacs formula, so if it doesn't work - it should be reported to Emacs devs.
